test/index.js

```diff
@@ -188,79 +189,13 @@ describe('big-json', function() {
             parseStream.on('error', function(err) {
                 assert.ok(err);
                 assert.equal(err.name, 'Error');
-                assert.equal(err.message, 'Parser has expected a string value');
+                assert.equal(err.message, 'Invalid JSON (Unexpected "\\n" ' +
+                    'at position 27 in state STOP)');
                 return done();
             });
 
             readStream.pipe(parseStream);
         });
-
-
-        it('should handle multibyte keys and vals', function(done) {
```

Comment:
Why are we dropping support for multibyte characters?

Reply:
Supported by default now (mentioned in OP :)).

Reply:
Shouldn't we keep the test to ensure future changes don't break multibyte?

Reply:
Readded